setup.cfg

```
@@ -73,13 +70,13 @@ include = torchgeo*
 # Optional dataset requirements
 datasets =
     h5py
-    # laspy 2+ required for Python 3.6+ support
+    # laspy 2+ required for Python 3.7+ support
     laspy>=2
     # open3d 0.11.2+ required to avoid GLFW error:
     # https://github.com/isl-org/Open3D/issues/1550
     open3d>=0.11.2
     opencv-python
-    # pandas 0.19.1+ required for python 3.6 support
+    # pandas 0.19.1+ required for python 3.7 support
     pandas>=0.19.1
```

Is an explicit minimum pin required here? If so, it should be set to pandas 1.2 for Python 3.7+, see https://github.com/pandas-dev/pandas/pull/35214
```suggestion
    # pandas 1.2.0+ required for python 3.7 support
    pandas>=1.2.0
```

---

> Is an explicit minimum pin required here?

I like to specify a minimum version for every dependency to prevent issues like #481, #438, #350, etc. In this case, since the constraint is imposed by supported Python versions instead of required features, it isn't strictly necessary, but I think it's still useful. Most Python libraries specify minimum supported versions but not maximum. So even if an old version of pandas doesn't support Python 3.7, it's still possible to install it in Python 3.7. This helps prevent that.

> If so, it should be set to pandas 1.2 for Python 3.7+, see pandas-dev/pandas#35214

pandas 1.2 was the first release to drop Python 3.6 support, but it wasn't the first release to add Python 3.7 support. According to the [release notes](https://pandas.pydata.org/docs/whatsnew/v0.23.2.html), pandas 0.23.2 is the first version to support Python 3.7. I'll update this.